src/main/java/classes/Drinks.java

```diff
@@ -0,0 +1,72 @@
+package classes;
+
+import java.util.LinkedList;
+
+public class Drinks 
+{
+    // attributes
+    // LinkedList data for all the Drink items
+    private LinkedList<Drink> allDrinks = new LinkedList<Drink>();
+
+    /*
+     * Constructor Name : Drinks()
+     * Parameters : none
+     * Description : Default constructor
+     */
+    public Drinks() 
+    {
+        drinkSeeder();
+    }
+    
+    /*
+     * Method Name : drinkSeeder()
+     * Parameters : none
+     * Description : - create all drink objects
+     */
+    private void drinkSeeder()
+    {
+        allDrinks.add(new Drink("Saffron Tea", 3.99, 4, Category.HOT, "Bread"));
```

Comment:
what is the Bread value ?
I saw that also in Foods, if it is default value, change it to an Empty string

Reply:
Bread is the pairing, its like pita bread

Reply:
Aha, then it is fine